src/agr/orthology_file_generator.py

```diff
@@ -0,0 +1,79 @@
+import sys
+
+import logging
+
+from dateutil.parser import parse
+from datetime import datetime
+from time import gmtime, strftime
+
+from neo4j.v1 import GraphDatabase
+
+logger = logging.getLogger(name=__name__)
+
+class OrthologyFileGenerator:
+
+    file_header_template = """#########################################################################
+#
+# Ortholog File
+# Source: Alliance of Genome Resources (Alliance)
+# Filter: stringent
+# Filter Details:
+#   Ortholog needs to be called by at least 3 Algorithms
+#        or is being called by either ZFIN or HGNC algorithms
+#        and either is best score or is best reverse score
+#   or ortholog is called by 2 algorithms and is best score and best reverse score
+# Datebase Version: {databaseVersion}
+# Date: {datetimeNow}
+#
+#########################################################################
+"""
+
+    def __init__(self, orthologs, generated_files_folder, database_version):
+        self.orthologs = orthologs
+        self.database_version = database_version
+        self.generated_files_folder = generated_files_folder
+
+    @classmethod
+    def _generate_header(cls, database_version):
+        return cls.file_header_template.format(datetimeNow = strftime("%Y-%m-%d %H:%M:%S", gmtime()),
```

Comment:
v. minor style nit pick - kwargs passed in calls should have no spaces around the `=` operator.